src/containers/Sponsors.jsx

```diff
@@ -28,6 +28,9 @@ export default class Sponsors extends React.Component{
                 <a href="https://www.ssmu.ca" target="_blank" rel="noopener noreferrer">
                   <Sponsor tier="P" year="2022" logo="ssmu.png"/>
                 </a>
+                <a href="https://www.pfizer.com" target="_blank" rel="noopener noreferrer">
```

Comment:
Could you actually reorder the sponsors in order of the amount they give? I believe SSMU is giving the most, and then pfizer, medtech. The other two are giving the least and the same amount I believe.